packages/forklift-console-plugin/locales/en/plugin__forklift-console-plugin.json

```diff
@@ -74,7 +74,7 @@
   "Clear all filters": "Clear all filters",
   "Click the pencil for setting provider web UI link": "Click the pencil for setting provider web UI link",
   "Click the update credentials button to save your changes, button is disabled until a change is detected.": "Click the update credentials button to save your changes, button is disabled until a change is detected.",
-  "Click the update hooks button to save your changes, button is disabled until a change is detected.": "Click the update hooks button to save your changes, button is disabled until a change is detected.",
+  "Click the update hooks button to save your changes, button is disabled until a change is detected or when the plan status does not enable editing.": "Click the update hooks button to save your changes, button is disabled until a change is detected or when the plan status does not enable editing.",
```

Comment:
Do you think it would make sense to display different messages based on the reason the button is disabled instead of explaining both sets of criteria to them since we know when it's disabled due to a status. Ex: `Plans in <status> status are not editable` or something like that.

Reply:
Done.

####  Screenshots:
1) 
![Screenshot from 2024-12-02 21-40-18](https://github.com/user-attachments/assets/dca0e215-4b5a-4048-86b9-8650affd9e41)



2)
![Screenshot from 2024-12-02 21-40-36](https://github.com/user-attachments/assets/13f83842-f7ce-4c29-a5c7-bf6a2f01ec9b)



  
3)
![Screenshot from 2024-12-02 21-43-50](https://github.com/user-attachments/assets/44ffed87-3e54-4272-be7f-70f1280b8967)